lib/cisco_node_utils/cmd_ref/span_session.yaml

```diff
@@ -0,0 +1,64 @@
+# SPAN session
+---
+_template:
+  nexus:
+    get_command: "show running monitor all"
+
+all_sessions:
+  multiple:
+  get_context: ~
+  get_value: '/^monitor session (\d+)$/'
```

Comment:
Don't add the `$` anchor to the end of this line, otherwise it will not match cases where additional information exists besides just the session_id.

```
monitor session 1 type local
monitor session 7 type erspan-destination
```